src/Rocketlet.ts

```diff
@@ -67,7 +69,9 @@ export abstract class Rocketlet {
      *
      * @return boolean stating whether the Rocketlet should be marked as active or not.
      */
-    public abstract initialize(): void;
+    public initialize(configurationExtend: IConfigurationExtend): void {
+        this.extendConfiguration(configurationExtend);
```

Comment:
In order to sequence default implementations, I implemented the generic method and made the more specific one(s) protected